static/mfb/mfb.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * Material floating button
+ * By: Nobita
+ * Repo and docs: https://github.com/nobitagit/material-floating-button
+ *
+ * License: MIT
+ */
+
+ // build script hook - don't remove
+ ;(function ( window, document, undefined ) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["undefined" is defined but never used](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2651071250/issues/source?bid=2452379&fileBranchId=2967311#l10)